storage/rocksdb/ha_rocksdb.cc

```diff
@@ -106,6 +106,7 @@ static st_io_stall_stats io_stall_stats;
 const std::string DEFAULT_CF_NAME("default");
 const std::string DEFAULT_SYSTEM_CF_NAME("__system__");
 const std::string PER_INDEX_CF_NAME("$per_index_cf");
+
```

Comment:
Spurious change?

Reply:
This is an upstream change so I prefer to keep it to minimize a diff with upstream.

Reply:
ack